.idea/misc.xml

```diff
@@ -2,6 +2,7 @@
 <project version="4">
   <component name="ExternalStorageConfigurationManager" enabled="true" />
   <component name="FrameworkDetectionExcludesConfiguration">
+    <file type="web" url="file://$PROJECT_DIR$/backend" />
```

Comment:
Does not IntelliJ want to put that there all the time on your machine?

Reply:
Yes but I think that causes intellij to only recognize the backend as a module and not recognize the flutter project. I have to rollback that line all the time, too...

Reply:
I think intellij has problems independent of this line... I checked the branch out and it works for me right now... Until intellij decides otherwise I gues...

Reply:
Okay, I removed the line.
Never again IntelliJ files in a repo 😅